src/v/serde/serde.h

```diff
@@ -533,17 +530,6 @@ inline void write(iobuf& out, iobuf t) {
     out.append(t.share(0, t.size_bytes()));
 }
 
-template<typename T>
-void write(iobuf& out, T t) {
-    using Type = std::decay_t<T>;
-    static_assert(
-      !is_chrono_time_point<Type>,
```

Comment:
Can we have an overload that `requires is_chrono_time_point`, and asserts with the same message, please. (we'd like people to think about it rather than just implement it).

Reply:
I added an overload for `std::chrono::time_point` which immediately `static_assert(false, ...)` with the same message as before.